integration_tests/async_integration_test.go

```diff
@@ -245,6 +245,9 @@ var _ = Describe("AsyncIntegrationTests", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(credsMap["uri"]).To(ContainSubstring("postgres"))
 
+			// The bind oepration finishes before the instance has finished updating
+			time.Sleep(60 * time.Second)
```

Comment:
I know it's way more work but did you consider fetching the Operation and polling? This [API](https://cloud.google.com/resource-manager/reference/rest/v1/operations/get) allows you to find a pending operation.

tests relying on time.Sleep() are likely to break

Reply:
That's a good idea, thanks!